### ISSUE TITLE
Implement new framework?

### ISSUE THREAD

**Issue by jssmith73** [jssmith73/Smith-Signature-Events#3]
Was thinking about using a new framework just to learn.

**Comment by thefrenchman1** [comment]
like what, python? or more react? angular?     I remember seeing a lot of Ruby on rails in applications

**Comment by jssmith73** [comment]
Yeah I was looking into Angular. Had to update Node.js first, which was a bitch. Ended up uninstalling just to redownload the latest version because I spent too much time working around issues with simply updating.

**Comment by jssmith73** [comment]
Spent a lot of time researching Angular before realizing there is a difference between Angular and Angular.js. Angular.js is what I'm using.
